.jenkins

```diff
@@ -1,17 +1,17 @@
 #!/usr/bin/env groovy
 
 // Include this shared CI repository to load script helpers and libraries.
-library identifier: 'vapor@1.0.0-RC0', retriever: modernSCM([
+library identifier: 'vapor@1.0.0-RC4', retriever: modernSCM([
```

Comment:
This is likely fine, but you may want to bump to 1.0.0-RC5 to get the tagging fix that landed this morning

Reply:
Nevermind, I see all the fun stuff is skipped. :+1: